public/client.js

```diff
@@ -1,9 +1,30 @@
-// $(function () {
-// 	$.get('/trucks', function (trucks) {
-// 		var truckList = [];
-// 		trucks.forEach(function (truck) {
-// 			truckList.push('<li><a href="/trucks/' + truck.name + '/">' + truck.name + '</a></li>')
-// 		});
-// 		$('.truck-list').append(truckList);
-// 	});
-// });
\ No newline at end of file
+ $(function () {
+ 	$.get('/trucks', function (trucks) {
+ 		var truckList = [];
+ 		trucks.forEach(function (truck) {
+ 			truckList.push('<li><a href="/trucks/' + truck.name + '/">' + truck.name + '</a></li>')
+ 		});
+ 		$('.truck-list').append(truckList);
+ 	});
+     
+     
+//     i tried the extra credit as follow, but it's not working
+     
+//    $('#myDiv').click(function() {
+//        $('.truck-list').hide();
+//        
+//        $.ajax({
+//            method: 'GET',
+//            url: '/food-types',
+//            dataType: JSON,
```

Comment:
JSON here needs to be a string value. JSON without quotes around it will be interpreted as a variable which is never defined, so it's the equivalent of saying `dataType: undefined,`